python/cudf/cudf/core/byte_pair_encoding.py

```diff
@@ -25,7 +25,9 @@ class BytePairEncoder:
     """
 
     def __init__(self, merges_pair: "cudf.Series"):
-        self.merge_pairs = cpp_merge_pairs(merges_pair._column)
+        self.merge_pairs = cpp_merge_pairs(
+            merges_pair._column.to_pylibcudf(mode="read")
+        )
```

Comment:
This looks odd because `cpp_merge_pairs` is being imported from the legacy Cython file, but that's just a transitive cimport of something from pylibcudf that is actually a class, not a function. Can we change to importing BPEMergePairs from pylibcudf and using that directly?

Reply:
Yes, done.